src/gfx/Stats.js

```diff
@@ -2,39 +2,35 @@
 
 import utils from '../utils';
 
-var now = utils.Timer.now;
+const now = utils.Timer.now;
 
 function createElement(tag, id, css) {
-  var element = document.createElement(tag);
+  const element = document.createElement(tag);
   element.id = id;
   element.style.cssText = css;
   return element;
 }
 
-function Stats() {
-  this.domElement = createElement('div', 'stats', 'padding:8px');
-  this._text = createElement('p', 'fps', 'margin:0;color:silver;font-size:large');
-  this.domElement.appendChild(this._text);
+class Stats {
+  constructor() {
+    this.domElement = createElement('div', 'stats', 'padding:8px');
+    this._text = createElement('p', 'fps', 'margin:0;color:silver;font-size:large');
+    this.domElement.appendChild(this._text);
 
-  this._startTime = now();
-  this._prevTime = this._startTime;
-
-  this._deltas = new Array(20);
-  this._index = 0;
-  this._total = 0.0;
-  this._count = 0;
-}
+    this._startTime = now();
+    this._prevTime = this._startTime;
 
-Stats.prototype = {
-  constructor: Stats,
+    this._deltas = new Array(20);
+    this._index = 0;
+    this._total = 0.0;
+    this._count = 0;
+  }
 
-  begin: function() {
-    this._startTime = now();
-  },
+  //begin() { this._startTime = now(); }
```

Comment:
Is there a reason to remove this method?

Reply:
Yes, this method is unused and its realization is in the constructor.